generator/build.gradle

```diff
@@ -20,7 +20,8 @@ dependencies {
     compile "com.google.inject:guice:${GUICE_VERSION}"
     compile "dk.brics.automaton:automaton:${AUTOMATON_VERSION}"
     compile "org.apache.commons:commons-csv:${COMMONS_CSV_VERSION}"
-    
+
+    testCompile project(":common").sourceSets.test.output
```

Comment:
does this need the .sourceSets.test.output?

Reply:
Yes otherwise it doesn't import the test src folder just the main one 👎